src/wasm/wasm-emscripten.cpp

```diff
@@ -482,13 +482,9 @@ std::string EmscriptenGlueGenerator::generateEmscriptenMetadata(
 
   meta << "  \"tableSize\": " << wasm.table.initial.addr << ",\n";
 
-  if (!initializerFunctions.empty()) {
+  if (initializer.is()) {
     meta << "  \"initializers\": [";
-    commaFirst = true;
-    for (const auto& func : initializerFunctions) {
-      meta << nextElement();
-      meta << "\"" << func.c_str() << "\"";
-    }
+    meta << "\n    \"" << initializer.c_str() << "\"";
```

Comment:
Do you plan to update the metadata format to make this not be a list any more as well?

Reply:
TBH I think we can completely remove this .. but we need to remove it from the emscripten side first.